pennylane_lightning_gpu/src/bindings/Bindings.cpp

```diff
@@ -85,6 +88,38 @@ void StateVectorCudaManaged_class_bindings(py::module &m) {
             return new StateVectorCudaManaged<PrecisionT>(
                 data_ptr, static_cast<std::size_t>(arr.size()));
         }))
+        .def(
+            "setZeroState",
+            [](StateVectorCudaManaged<PrecisionT> &sv, const size_t index,
+               const bool use_async) {
+                std::complex<PrecisionT> value(0, 0);
+                sv.setState(value, index, use_async);
+            },
+            "Create basis state on GPU.")
+        .def(
+            "setBasisState",
+            [](StateVectorCudaManaged<PrecisionT> &sv, const size_t index,
+               const bool use_async) {
+                std::complex<PrecisionT> value(1, 0);
+                sv.setState(value, index, use_async);
+            },
+            "Create basis state on GPU.")
```

Comment:
Do we need `setZeroState` if we have this? Also, `setZeroState` just 0's the memory, and doesn't create a `|0...0>` state.